### PR TITLE
Added table of content to Readme.md

### PR DESCRIPTION
**Pull request by simonv3**
I ran `doctoc` on README.md and it generated the table of content.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,37 @@
...
+  - [Contributing](#contributing)
+  - [Contributor Developer Setup](#contributor-developer-setup)
+    - [Run local urigo:angular in your project](#run-local-urigoangular-in-your-project)
+    - [Running tests](#running-tests)
+    - [Contributing to documentation and tutorials.](#contributing-to-documentation-and-tutorials)
+  - [Usage](#usage)
+    - [Table of Contents](#table-of-contents)
+    - [App initialization](#app-initialization)
+    - [Templating](#templating)
+- [Hello {{yourName}}!](#hello-yourname)
```
Comment (Urigo) [inline README.md]: @simonv3 is that working?

Comment (Urigo) [inline README.md]: The {{your name}} part inside markdown?

Comment (simonv3) [inline README.md]: @Urigo I think so! I suspect that github probably escapes anything before it displays the markdown. I've seen it in other places I think (though don't ask me where) - I also just tried it in the comment box here and clicked on preview:

[Hello {{yourname}}!](#hello-yourname)

Comment (Urigo) [inline README.md]: I only see it as Hello {{yourname}} .....

Comment (simonv3) [inline README.md]: Oh! I thought that was intended. Hmmm - I see what's going on. It identified the <h1> tag in the code block of the markdown here: https://github.com/Urigo/angular-meteor#templating as an actual h1 tag. Interesting. That seems like a bug in doctoc.

Comment (Urigo) [inline README.md]: Ohh funny :)

**Comment by Urigo** [comment]
@simonv3 can I merge it now?

**Comment by simonv3** [comment]
@Urigo I think so! Running doctoc will have the same problem in the future though, so just gotta keep that in mind.

**Pull request merged**